Dockerfile.fluentbit

```diff
@@ -1,9 +1,131 @@
+    -DFLB_CUSTOM_CALYPTIA="Off" \
+    -DFLB_DEBUG="Off" \
+    -DFLB_EXAMPLES="Off" \
+    -DFLB_HTTP_SERVER="On" \
+    -DFLB_LUAJIT="Off" \
+    -DFLB_METRICS="Off" \
+    -DFLB_PROXY_GO="Off" \
+    -DFLB_RELEASE="On" \
+    -DFLB_SIGNV4="Off" \
+    -DFLB_SHARED_LIB="Off" \
+    -DFLB_STREAM_PROCESSOR="Off" \
+    -DFLB_WAMRC="Off" \
+    -DFLB_WASM="Off" \
+    -DFLB_FILTER_AWS="Off" \
+    -DFLB_FILTER_CHECKLIST="Off" \
+    -DFLB_FILTER_ECS="Off" \
+    -DFLB_FILTER_EXPECT="Off" \
+    -DFLB_FILTER_GEOIP2="Off" \
+    -DFLB_FILTER_KUBERNETES="Off" \
+    -DFLB_FILTER_LUA="Off" \
+    -DFLB_FILTER_NIGHTFALL="Off"\
+    -DFLB_FILTER_RECORD_MODIFIER="Off" \
+    -DFLB_FILTER_STDOUT="Off" \
+    -DFLB_FILTER_THROTTLE_SIZE="Off" \
+    -DFLB_FILTER_THROTTLE="Off" \
+    -DFLB_FILTER_WASM="Off" \
+    -DFLB_IN_COLLECTD="Off" \
+    -DFLB_IN_CPU="Off" \
+    -DFLB_IN_DISK="Off" \
+    -DFLB_IN_DOCKER_EVENTS="Off" \
+    -DFLB_IN_DOCKER="Off" \
+    -DFLB_IN_DUMMY="Off" \
+    -DFLB_IN_EXEC_WASI="Off" \
+    -DFLB_IN_EXEC="Off" \
+    -DFLB_IN_FLUENTBIT_METRICS="Off" \
+    -DFLB_IN_FORWARD="Off" \
+    -DFLB_IN_HEAD="Off" \
+    -DFLB_IN_HEALTH="Off" \
+    -DFLB_IN_HTTP="Off" \
+    -DFLB_IN_KMSG="Off" \
+    -DFLB_IN_MEM="Off" \
+    -DFLB_IN_MQTT="Off" \
+    -DFLB_IN_NETIF="Off" \
+    -DFLB_IN_NGINX_EXPORTER_METRICS="Off" \
+    -DFLB_IN_NODE_EXPORTER_METRICS="Off" \
+    -DFLB_IN_PROC="Off" \
+    -DFLB_IN_PROMETHEUS_SCRAPE="Off" \
+    -DFLB_IN_RANDOM="Off" \
+    -DFLB_IN_SERIAL="Off" \
+    -DFLB_IN_STATSD="Off" \
+    -DFLB_IN_STDIN="Off" \
+    -DFLB_IN_SYSLOG="Off" \
+    -DFLB_IN_SYSTEMD="On" \
+    -DFLB_IN_TCP="Off" \
+    -DFLB_IN_THERMAL="Off" \
+    -DFLB_IN_OPENTELEMETRY="Off" \
+    -DFLB_IN_UDP="Off" \
+    -DFLB_OUT_AZURE_BLOB="Off" \
+    -DFLB_OUT_AZURE_KUSTO="Off" \
+    -DFLB_OUT_AZURE="Off" \
+    -DFLB_OUT_BIGQUERY="Off" \
+    -DFLB_OUT_CALYPTIA="Off" \
+    -DFLB_OUT_CLOUDWATCH_LOGS="Off" \
+    -DFLB_OUT_COUNTER="Off" \
+    -DFLB_OUT_DATADOG="Off" \
+    -DFLB_OUT_ES="Off" \
+    -DFLB_OUT_EXIT="Off" \
+    -DFLB_OUT_FILE="Off" \
+    -DFLB_OUT_FLOWCOUNTER="Off" \
+    -DFLB_OUT_GELF="Off" \
+    -DFLB_OUT_HTTP="Off" \
+    -DFLB_OUT_INFLUXDB="Off" \
+    -DFLB_OUT_KAFKA_REST="Off" \
+    -DFLB_OUT_KAFKA="Off" \
+    -DFLB_OUT_KINESIS_FIREHOSE="Off" \
+    -DFLB_OUT_KINESIS_STREAMS="Off" \
+    -DFLB_OUT_LIB="Off" \
+    -DFLB_OUT_LOGDNA="Off" \
+    -DFLB_OUT_LOKI="Off" \
+    -DFLB_OUT_NATS="Off" \
+    -DFLB_OUT_NRLOGS="Off" \
+    -DFLB_OUT_OPENSEARCH="Off" \
+    -DFLB_OUT_OPENTELEMETRY="Off" \
+    -DFLB_OUT_PGSQL="Off" \
+    -DFLB_OUT_PLOT="Off" \
+    -DFLB_OUT_PROMETHEUS_EXPORTER="Off" \
+    -DFLB_OUT_PROMETHEUS_REMOTE_WRITE="Off" \
+    -DFLB_OUT_S3="Off" \
+    -DFLB_OUT_SKYWALKING="Off" \
+    -DFLB_OUT_SLACK="Off" \
+    -DFLB_OUT_SPLUNK="Off" \
+    -DFLB_OUT_STACKDRIVER="Off" \
+    -DFLB_OUT_STDOUT="Off" \
+    -DFLB_OUT_SYSLOG="Off" \
+    -DFLB_OUT_TD="Off" \
+    -DFLB_OUT_WEBSOCKET="Off"
```

Comment:
are all these options required, or is this *just* to be more explicit with how we build the image?

Reply:
In v2 (not v1, but, uhhh, v2 would be nice later) those are all "On" by default.